archimedes/archimedes/functions/list/manipulations.py

```diff
@@ -19,6 +19,9 @@ def _flatten(l):
                 yield el
     return list(_flatten(ls))
 
+def reverse(ls: list):
```

Comment:
nit: is this type hint wrong? Should it be something like `List[str]`?

Reply:
For this function, the list can be of anything so I don't think it makes sense to use any content-typing restriction like that. (`list` and `List[<type>]` are both real options for python typing, just the latter lets you additionally hint at typing of the contents.)

Reply:
Ah, got it! I thought `list` was only the invocation for a new list. Thanks!